dev-client/src/components/buttons/icons/IconButton.tsx

```diff
@@ -59,28 +59,31 @@ export const IconButton = forwardRef<View, IconButtonProps>(
     ref,
   ) => {
     return (
-      <Pressable
-        ref={ref}
-        accessibilityRole="button"
-        accessibilityLabel={accessibilityLabel}
-        accessibilityState={{disabled: disabled}}
-        disabled={disabled}
-        style={[
-          containerStyleForType(type),
-          containerStyleForVariant(variant, disabled),
-        ]}
-        onPress={onPress}>
-        <MaterialIcon
-          name={name}
+      <View>
```

Comment:
This is a fix to a styling problem that I uncovered in my `BaseButton` implementation, but also affected this component.